Zapper-fi/studio
# fix(vvs-finance): add labels and xVVS token balance

## Description

* add xVVS app token balance
* add labels for auto vvs mine, farmsV2, farms, manual vvs mine (within farms) and mines

## Checklist

- [x] I have followed the [Contributing Guidelines](https://github.com/Zapper-fi/studio/blob/edbdd22/CONTRIBUTING.md)
- [x] (optional) As a contributor, my Twitter handle is: https://twitter.com/cronos_labs

## How to test?

1. edit/create `.env` adding `ENABLED_APPS="vvs-finance"`
2. run `pnpm dev` to spin up nest server
3. visit dev endpoints below to see result of fetchers to integrate vvs-finance:
  * http://localhost:5001/apps/vvs-finance/positions?groupIds[]=auto-vvs-mine&network=cronos
  * http://localhost:5001/apps/vvs-finance/positions?groupIds[]=farm-v2&network=cronos
  * http://localhost:5001/apps/vvs-finance/positions?groupIds[]=farm&network=cronos
  * http://localhost:5001/apps/vvs-finance/positions?groupIds[]=mine&network=cronos
  * `http://localhost:5001/apps/vvs-finance/balances?addresses[]=<wallet_address>&network=cronos`
